README.md

```diff
@@ -222,12 +222,12 @@ If [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy), the modul
 If a value of `'minimal'` is set, the progress indicator will render as a small, colored bar at the top of the window. This can be useful when the default fancy progress indicator interferes with elements in the page.
 
 ### `static`
-Type: `String | Array(String)`<br>
+Type: `String | Array(String) | Object`<br>
```

Comment:
Can we give concrete examples of the usage below? Now it's in the paragraph and you have to dig for the information. Having a couple would clarify the usage a lot as then people can scan instead of reading.

Reply:
yeah I suppose I can break it out into a block